README.md

```
@@ -239,4 +239,4 @@ For issues, questions, or contributions, please:
 
 ---
 
-Happy Performance Testing! 🎯
+Happy Performance Testing! 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)